site/fetchers/badger-brain.js

```diff
@@ -64,16 +64,27 @@ const fullEventsQuery = `
   ${dateTimeFieldsEvents}
 `;
 
-export function getEvents() {
+export function getData() {
   const body = `
     query {
       allEvents {
         ${fullEventsQuery}
       }
+      allBadgers {
+        firstName
+        lastName
+        jobTitle
+        startDate
+        imageUrl
+        categories
+      }
     }
   `;
 
   return fetch(badgerBrainEndpoint, getRequestOptions(body))
           .then(response => response.json())
-          .then(events => sortEvents(selectValidEvents(events.data.allEvents)));
+          .then(({ data }) => ({
+            events: sortEvents(selectValidEvents(data.allEvents)),
+            badgers: data.allBadgers,
```

Comment:
Agreed. I've moved `getCategories` out of `state.index.js` into here so that it matches `selectValidEvents`